doc/mgr/administrator.rst

```diff
@@ -53,6 +53,27 @@ by a standby.
 If you want to pre-empt failover, you can explicitly mark a ceph-mgr
 daemon as failed using ``ceph mgr fail <mgr name>``.
 
+Performance
+-----------
+
+All the mgr modules share a cache that can be enabled with
+``ceph config set mgr mgr_ttl_cache_expire_seconds <seconds>``, where seconds
+is the time to live of the cached python objects.
+
+The default behaviour of a mgr daemon consists of sending python objects with
+data to the mgr modules. When you enable the cache the default python object
+formatter is replaced to a json serializator.
+
+It is recommended to enable the cache when there are 500+ osds or 10k+ pgs
+as internal structures might increase in size, and cause latency
+issues when requesting large structures. As an example, an OSDMap with 1000 osds
+has a aproximate size of 4MiB. With heavy load, on a 3000 osd cluster there has
+been a 1.5x improvement enabling the cache.
+
+Furthermore, you can run ``ceph config-key get "mgr/cache/cache_hit_count`` and
+``ceph config-key get "mgr/cache/cache_miss_count`` to retrieve the hit/miss ratio
+of the cache.
```

Comment:
Are we doing this? We shouldn't use ceph-mon KV entries to store rapidly changing data. If we need these data we can/should use perf-counters instead.

Reply:
I needed something to store the hit-miss ratio to add integration tests and I didn't know how perf counters worked. How are they done?